### PR TITLE
[TD-11773]<feature>(tools): taosbenchmark support taosadapter

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: tests/develop-test/5-taos-tools/taosbenchmark/json/rest_auto_create_table.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "port": 6030,
   "user": "root",
   "password": "taosdata",
-  "thread_count": 4,
-  "thread_pool_size": 20,
+  "thread_count": 1,
```
Comment (sangshuduo) [inline tests/develop-test/5-taos-tools/taosbenchmark/json/rest_auto_create_table.json]: how about using multithread?

File: tests/develop-test/5-taos-tools/taosbenchmark/json/sml_auto_create_table.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "port": 6030,
   "user": "root",
   "password": "taosdata",
-  "thread_count": 4,
-  "thread_pool_size": 20,
+  "thread_count": 1,
```
Comment (sangshuduo) [inline tests/develop-test/5-taos-tools/taosbenchmark/json/sml_auto_create_table.json]: how about using multithread?

File: tests/develop-test/5-taos-tools/taosbenchmark/json/sml_insert_alltypes.json [hunk 3]
```diff
@@ -5,8 +5,8 @@
   "port": 6030,
   "user": "root",
   "password": "taosdata",
-  "thread_count": 4,
-  "thread_pool_size": 20,
+  "thread_count": 1,
```
Comment (sangshuduo) [inline tests/develop-test/5-taos-tools/taosbenchmark/json/sml_insert_alltypes.json]: same question

File: tests/develop-test/5-taos-tools/taosbenchmark/json/sml_json_alltypes.json [hunk 4]
```diff
@@ -5,8 +5,8 @@
   "port": 6030,
   "user": "root",
   "password": "taosdata",
-  "thread_count": 4,
-  "thread_pool_size": 20,
+  "thread_count": 1,
```
Comment (sangshuduo) [inline tests/develop-test/5-taos-tools/taosbenchmark/json/sml_json_alltypes.json]: same question

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comments. And 2.4 needs a PR too.

**Review by afwerar — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**